src/api/config/block-sync-params.md

```diff
@@ -0,0 +1,27 @@
+# Block Sync Parameters
+
+TODO Explain module
+
+## `block_sync.block_batch_amount`
+
+- **Type:** Number
+- **Default:** $4$
+
+The number of blocks that can be sent in one message.
```

Comment:
I wonder why the explanation of a config parameter goes after its specifications (type, values, default value, etc.). Why not change this order?

Reply:
I have no objections, and you probably understand better how it would be more comprehensible. Should I change the order everywhere?